attributes/default.rb

```diff
@@ -1,7 +1,7 @@
 #### Install Info
-default["eucalyptus"]["install-type"] = "packages"
```

Comment:
Hey @mbacchi We need to keep this at packages as that is the default for installs outside of eucadev. The other changes are spot on though.